crossplane/aws/README.md

```diff
@@ -0,0 +1,77 @@
+# Crossplane Environment Composition for platform building
+
+This tutorial focus on creating a basic AWS EKS cluster
+
+This Crossplane Composite resource creates the following resources:
+- EKS Cluster
+- NodeGroup
+<!-- - Helm Provider Config
+- Helm Release inside the created cluster -->
+
+## Installation
+
+In an EKS Kubernetes Cluster install the following components.
+
+Let's install [Upbound's Crossplane CLI](https://docs.upbound.io/cli/):
+
+```
+curl -sL "https://cli.upbound.io" | sh
+
+```
+
+Download [Upbound Universal Crossplane](https://docs.upbound.io/uxp/install/?ref=upbound-blog#install-upbound-universal-crossplane)
+
+```
+# Make sure your ~/.kube/config file points to your cluster
+
+up uxp install
```

Comment:
@asarenkansah why are you using this? Can we just not rely on Crossplane upstream? Why is this different?

Reply:
No particular reason, I chose upbound because they have good aws compositions curated for EKS. I leveraged them and made modifications to fit this specific use case. I also like the registry they have

I can switch it all over to the upstream crossplane community provider if you'd prefer that - will just take a bit more time

Reply:
I will need to take a look at how would that work.. I don't like to promote company specific things. 
I don't understand about the exact differences between the upbound compositions and the upstream ones. Don't change this for now.. I will do my research first.